lib/libzlib/sub.mk

```diff
@@ -1,11 +1,11 @@
 global-incdirs-y += include
 
+cflags-y += -Wno-extra
 cflags-y += -Wno-old-style-definition
 cflags-y += -Wno-switch-default
 cflags-y += -Wno-strict-prototypes
 cflags-y += -Wno-missing-prototypes
 cflags-y += -Wno-missing-declarations
-cflags-y += -Wno-shift-negative-value
```

Comment:
Are warnings about shifting negative values  disabled by default in GCC7? (or I guess they are included in extra warning, right?)